### PR TITLE
[AIRFLOW-5149] - skip SLA checks config

### PR DESCRIPTION
**Pull request by tooptoop4**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### CODE REVIEW

**Comment by tooptoop4** [comment]
travis is unrelated?

File: airflow/jobs/scheduler_job.py [hunk 1]
```diff
@@ -715,7 +715,8 @@ def _process_dags(self, dagbag, dags, tis_out):
                             schedule_delay)
                 self.log.info("Created %s", dag_run)
             self._process_task_instances(dag, tis_out)
-            self.manage_slas(dag)
+            if conf.getboolean('core', 'CHECK_SLAS'):
```
Comment (kaxil) [inline airflow/jobs/scheduler_job.py]:
```suggestion
            if conf.getboolean('core', 'CHECK_SLAS', fallback=True):
```

**Review by turbaszek — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

File: airflow/config_templates/default_airflow.cfg [hunk 2]
```diff
@@ -176,6 +176,9 @@ store_serialized_dags = False
 # Updating serialized DAG can not be faster than a minimum interval to reduce database write rate.
 min_serialized_dag_update_interval = 30
 
+# On each dagrun check against defined SLAs
```
Comment (mik-laj) [inline airflow/config_templates/default_airflow.cfg:179]: Can you add some documentation about this option?   If there is no information about this feature in the documentation, very few people will be able to use it.

Comment (potiuk) [reply airflow/config_templates/default_airflow.cfg:179]: @mik-laj - > where do you think the documentation should be added ? Maybe you can provide some pointers? Most of the config options are documented in comments of the default_airlfow.cfg. Any other specific place you think this documentation should be added (and where all the previous options are documented?)

Comment (mik-laj) [reply airflow/config_templates/default_airflow.cfg:179]: > and where all the previous options are documented?

We have documentation gaps, but we try to develop documentation all the time. There is no precise documentation for the SLA mechanism, but this does not mean that it is impossible to write a few sentences and put them in the documentation. Another person who wants to describe the wider documentation will have less work.

I think that these two sentences describing this parameter may be placed in a new document in the howto directory. I will give an example of documentation that describes the configuration parameter.
https://github.com/apache/airflow/pull/5850/files

Comment (kaxil) [reply airflow/config_templates/default_airflow.cfg:179]: Created at https://github.com/apache/airflow/pull/6974

Comment (mik-laj) [reply airflow/config_templates/default_airflow.cfg:179]: @kaxil Fantastic!

Comment (potiuk) [reply airflow/config_templates/default_airflow.cfg:179]: Great! Thanks @mik-laj and @kaxil for explaining and adding :).

**Pull request merged**